src/components/bar-chart/bar-chart.js

```diff
@@ -52,9 +82,17 @@ export class BarChartComponent {
     });
   }
 
+  makeZeroDataset() {
+    this.dataset = {
```

Comment:
you do not need this method -- you can just make this assignment in `constructor`